app/portainer/components/information-panel/informationPanel.html

```diff
@@ -6,7 +6,7 @@
           <span style="float: left;">
             {{ $ctrl.titleText }}
           </span>
-          <span class="small" style="float: right;">
+          <span class="small" style="float: right;" ng-if="!$ctrl.cantDismiss">
```

Comment:
You can simply relay on the fact that the `dismissAction` binding was specified here: `ng-if="$ctrl.dismissAction"`

Reply:
Changing to `ng-if="$ctrl.dismissAction"` doesn't hide the button even if the `dismissAction` binding isn't provided.

When using the `&` binding, angular wraps the function in order to keep references to the original $scope of the passed method, even if is not defined.
Function is :
```
ƒ (locals) {
   return parentGet(scope, locals);
}
```